Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.3
```

Comment:
Didn't change the Swift Language version for the targets to avoid the risk of making any code changes. Any concerns about using 5.3 only for Swift Package Manager?
Not enough experience with SPMs, so not sure if its acceptable.

Reply:
I wouldn't think that would be a problem but I'm not an SPM expert by any stretch of the imagination so 🤷